src/Octoshift/Services/VersionChecker.cs

```diff
@@ -61,7 +60,7 @@ public async Task<string> GetLatestVersion()
             }
             response.EnsureSuccessStatusCode();
 
-            _latestVersion = content.TrimStart('v', 'V').Trim();
+            _latestVersion = Version.Parse(content.TrimStart('v', 'V').Trim());
```

Comment:
In theory, this could error if the value in the file wasn't a valid version. This wouldn't break the CLI, because we have a `try/catch` in relevant places.